js/base/Exchange.js

```diff
@@ -1063,6 +1063,12 @@ module.exports = class Exchange {
         return this.implodeParams (url, { 'hostname': this.hostname })
     }
 
+    resolvePath (path, params) {
```

Comment:
Since the base classes are not transpiled, we should also add it to exchange.py and to Exchange.php.